frontend/src/components/custom-tools/combined-output/CombinedOutput.jsx

```diff
@@ -91,14 +102,14 @@ function CombinedOutput({ docId, setFilledFields }) {
       });
   }, [docId, singlePassExtractMode, isSinglePassExtractLoading]);
 
-  useEffect(() => {
-    Prism.highlightAll();
-  }, [combinedOutput]);
-
   const handleOutputApiRequest = async () => {
+    let url = `/api/v1/unstract/${sessionDetails?.orgId}/prompt-studio/prompt-output/?tool_id=${details?.tool_id}&document_manager=${docId}&is_single_pass_extract=${singlePassExtractMode}`;
```

Comment:
@tahierhussain What is this URL used for ? 
Suggestion: Is there a way to make it better by having a separate helper function for creating URL in case of future usability ?

Reply:
@tahierhussain Yeah seems like the right idea, the file changes itself will be large in the PR.
Can you add a ticket for this if you haven't already just to track. Thanks.

Reply:
@athul-rs Here's the ticket for this,
https://zipstack.atlassian.net/browse/UN-1435